### DISCUSSION TITLE
What all lua script/plugins have has everyone written

### DISCUSSION THREAD

**Discussion by bonelifer** [jcorporation/mympd-scripts#8] [General]
What all lua script/plugins has everyone written?

**Comment by jcorporation** [comment]
I'm curious to see what comes together!

**Comment by bonelifer** [comment]
Here's everything I've found, only one is lua/plugin: 

So far only found one, last.fm scrobbler:
https://github.com/loop333/mympd_lastfm_scrobbler

Also found Chinese language package from 2020:
https://github.com/timpani2012/myMPD-in-Chinese

And an ARCH Linux package build for mpd+flac and mympd as one package:
https://github.com/trclst/mpd-flac-mympd

Sandstorm port of mympd:
https://github.com/zenhack/mympd-sandstorm

Third party docker image for  unraid mpd+myMPD for Snapcast:
https://github.com/daredoes/docker-mympd

**Reply by jcorporation** [reply]
> So far only found one, last.fm scrobbler:
https://github.com/loop333/mympd_lastfm_scrobbler

This is now integrated in the official mympd-script repository.
